signprot/urls.py

```diff
@@ -1,28 +1,28 @@
-from contactnetwork.views import PdbTableData
-from django.conf.urls import url
+from . import views
+from django.urls import path, re_path
 from django.views.decorators.cache import cache_page
-from signprot import views
 from signprot.views import *
+from contactnetwork.views import PdbTableData
 
 urlpatterns = [
-    url(r'^$', views.BrowseSelection.as_view(), name='index'),
-    url(r'^arrestin$', views.ArrestinSelection.as_view(), name='arrestin'),
-    url(r'^statistics/(?P<dataset>[^/]*?)/$',  views.GProtein, name='gprotein'),
-    url(r'^statistics_venn',  views.GProteinVenn, name='gprotein'),
-    url(r'^statistics_tree',  views.GProteinTree, name='gprotein'),
-    url(r'^statistics',  views.GProtein, name='gprotein'),
-#    url(r'^couplings',  views.couplings, name='couplings'),
-    url(r'^couplings$', (CouplingBrowser.as_view()), name='coupling_browser'),
-    url(r'^ginterface/(?P<protein>[^/]*?)/$', views.Ginterface, name='render'),
-    url(r'^ginterface[/]?$', views.TargetSelection.as_view(), name='targetselection'),
-    url(r'^ajax/barcode/(?P<slug>[-\w]+)/(?P<cutoff>\d+\.\d{0,2})/$', views.ajaxBarcode, name='ajaxBarcode'),
-    url(r'^ajax/interface/(?P<slug>[-\w]+)/$', views.ajaxInterface, name='ajaxInterface'),
-    url(r'^structure/(?P<pdbname>[-\w]+)/$', views.StructureInfo, name='StructureInfo'),
-    url(r'^family/(?P<slug>[-\w]+)/$', views.familyDetail, name='familyDetail'),
-    url(r'^matrix[/]?$', views.InteractionMatrix, name='InteractionMatrix'),
-    url(r'^matrix/seqsig/', views.IMSequenceSignature, name='SequenceSignature'),
-    url(r'^matrix/sigmat/', views.IMSignatureMatch, name='SignatureMatch'),
-    url(r'^matrix/render_sigmat/$', views.render_IMSigMat, name='renderSignatureMatch'),
-    url(r'^pdbtabledata', PdbTableData, name='pdbtreedata'),
-    url(r'^(?P<slug>[-\w]+)/$', views.signprotdetail, name='signprotdetail'),
+    path('', views.BrowseSelection.as_view(), name='index'),
```

Comment:
Good point. I guess we should also convert some "re_path" to "path" calls.

Reply:
True. Better to convert re_path to path.
Just don't know how the "regexes" work with path alone yet. Will look into it.